root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -0,0 +1,69 @@
+package com.example.servingwebcontent;
+
+import java.sql.*;
+import java.util.*;
+
+public class ConnectDatabase {
+	Connection con = null;
+
+	public ConnectDatabase (String database, String user, String pass) {
+		try {
+			con = DriverManager.getConnection("jdbc:mysql://localhost/" + database + "?" + "user=" + user + "&password=" + pass);
+		} 
+		catch (SQLException e) { 
+			System.out.println(e);
+			throw new RuntimeException(); 
+		}
+	}
+
+	public ResultSet queryDB (String query) {
+		ResultSet rs = null;
+		try {
+			Statement stmt = con.createStatement();
+			rs = stmt.executeQuery(query);
+		} catch (SQLException e) { 
+			System.out.println(e);
+			this.closeConnection();
```

Comment:
maybe don't need this here. The code that calls this can take care of it if they want.